packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -67,6 +68,7 @@ export const NotifiCard: React.FC = () => {
           inputSeparators={intercomInputSeparators}
           cardId="1045f61752b148eabab0403c08cd60b2"
         />
+        <NotifiSubscriptionContextWrapper cardId="1045f61752b148eabab0403c08cd60b2"></NotifiSubscriptionContextWrapper>
```

Comment:
Can you please update this to 
`<NotifiSubscriptionContextWrapper cardId="1045f61752b148eabab0403c08cd60b2" />`

We probably shouldn't commit testing updates in notifi-react-example but I think it is also okay for now.

Reply:
i'll just back out the notfii example changes